libraries/core_libs/network/include/network/tarcap/packets_handlers/common/packet_handler.hpp

```diff
@@ -32,14 +34,31 @@ class PacketHandler {
   void processPacket(const PacketData& packet_data);
 
  private:
-  void handle_read_exception(const PacketData& packet_data);
+  void handle_caught_exception(const char* exception_msg, const PacketData& packet_data,
```

Comment:
this is ugly, can we used string_view here?

Reply:
uu very good point actually :)